0000-template.md

```diff
@@ -1,51 +1,48 @@
+- Title: (fill me in with a human-focussed title for the RFC)
 - Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
 - Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
-- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- RFC PR: [#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Tracking Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+- Team: (fill me in with the team or teams responsible for this RFC)
+- Keywords: (fill me in with keywords which may be useful when searching for this RFC)
+- Previous RFCs: [#0000](https://github.com/rust-lang/rfcs/pull/0000) (fill me in with RFC numbers for any RFCs this RFC supersedes, deprecates, or extends)
+- Previous discussion: (fill me in with links to previous discussions such as internals.rust-lang.org or Zulip threads, or issues)
 
 # Summary
 [summary]: #summary
 
 One paragraph explanation of the feature.
 
-# Motivation
-[motivation]: #motivation
+# Motivation and background
+[motivation-and-background]: #motivation-and-background
 
-Why are we doing this? What use cases does it support? What is the expected outcome?
+Why are we doing this? What use cases does it support? What is the expected outcome? Why are existing solutions not good enough? If possible, include data to support your claims. Include any background context useful for understanding the RFC.
 
-# Guide-level explanation
-[guide-level-explanation]: #guide-level-explanation
+# Detailed explanation
+[detailed-explanation]: #detailed-explanation
 
-Explain the proposal as if it was already included in the language and you were teaching it to another Rust programmer. That generally means:
+Explain the proposal in detail. Specify the proposal as it would be experienced by a user (usually a Rust developer) and so that it can be understood by an implementer. That generally means:
```

Comment:
Please disbiguate here, via some phrasing, between developers of Rust itself and developers writing code in Rust.